etc/config/compiler-explorer.amazon.properties

```diff
@@ -15,10 +15,15 @@ proxyRetries=10
 proxyRetryMs=500
 rescanCompilerSecs=3600
 sentryDsn=https://8e4614f649ad4e3faf3e7e8827b935f9@sentry.io/102028
-formatters=clangformat
+formatters=clangformat:rustfmt
```

Comment:
I think one big mistake I did with this is to make the formatter config live in the compiler-explorer properties file. 
I did it because it was easier, but now that there's going to be more than 1, I think we should try make them like tools and libs and have a formatters peroperty for each language, but I'm not sure if it's even doable.
As it is, this PR is great so maybe we want to first merge this, and then work in improving the config situation

Reply:
I'm not sure how easy it'll be to move everything out of CE props, I'll play around with it and see what I come up with.

If you'd like to merge it as-is and work on refactoring out later then that's also good with me. I'll mark it as ready.